plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -68,7 +69,6 @@
   private static final Logger LOG = LoggerFactory.getLogger(RocksDBColumnarKeyValueStorage.class);
   private static final String DEFAULT_COLUMN = "default";
   private static final String NO_SPACE_LEFT_ON_DEVICE = "No space left on device";
-  private static final int ROCKSDB_FORMAT_VERSION = 5;
```

Comment:
why this is removed?

Reply:
I think this is done by error, we should keep it. I will update this part, thanks.